swagger_to_sdk_config.json

```diff
@@ -0,0 +1,101 @@
+    },
+    "arm-datalake-analytics/catalog/2015-10-01-preview/swagger/catalog.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/dataLake.Analytics/lib/catalog"
+    },
+    "arm-datalake-store/account/2015-10-01-preview/swagger/account.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/dataLake.Store/lib/account"
+    },
+    "arm-datalake-store/filesystem/2015-10-01-preview/swagger/filesystem.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/dataLake.Store/lib/filesystem"
+    },
+    "arm-graphrbac/1.6-internal/swagger/graphrbac.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler Swagger",
+      "output_dir": "lib/services/graphManagement/lib"
+    },
+    "arm-intune/2015-01-14-preview/swagger/intune.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/intune/lib"
+    },
+    "arm-network/2016-03-30/swagger/network.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler Swagger",
+      "output_dir": "lib/services/networkManagement2/lib"
+    },
+    "arm-redis/2015-08-01/swagger/redis.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler Swagger",
+      "output_dir": "lib/services/rediscachemanagement/lib"
+    },
+    "arm-resources/resources/2016-02-01/swagger/resources.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/resourceManagement/lib/resource"
+    },
+    "arm-resources/subscriptions/2015-11-01/swagger/subscriptions.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/resourceManagement/lib/subscription"
+    },
+    "arm-resources/locks/2015-01-01/swagger/locks.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/resourceManagement/lib/lock"
+    },
+    "arm-resources/features/2015-12-01/swagger/features.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/resourceManagement/lib/feature"
+    },
+    "arm-resources/policy/2016-04-01/swagger/policy.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
+      "output_dir": "lib/services/resourceManagement/lib/policy"
+    },
+    "arm-storage/2016-01-01/swagger/storage.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 2 -Modeler Swagger",
+      "output_dir": "lib/services/storageManagement2/lib"
+    },
+    "servicefabric/2016-01-28/swagger/servicefabric.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -Modeler Swagger",
```

Comment:
@lmazuel 
This service needs `-CodeGenerator NodeJS` instead of "Azure.NodeJS". I assume the current script is adding  `-CodeGenerator Azure.NodeJS` by default. Is there a way for me to specify this?